app/src/main/java/org/jellyfin/androidtv/util/profile/deviceProfile.kt

```diff
@@ -280,13 +280,49 @@ fun createDeviceProfile(
 		}
 	}
 
-	// Limit video resolution support for older devices
-	codecProfile {
-		type = CodecType.VIDEO
+	// Get max resolutions for common codecs
+	// AVC
+	if(supportsAVC){
```

Comment:
These "supportsX" statements shouldn't be necessary for codec profiles. If the direct play profile doesn't include a codec it won't use it anyway.

Reply:
Removed the supportsX statements. I added them because I saw similar statements right before the ones I added.